dali/test/python/test_pipeline.py

```diff
@@ -324,6 +307,66 @@ def iter_setup(self):
             check_batch(outs[0], outs[2], batch_size)
             check_batch(outs[1], outs[3], batch_size)
 
+def test_cropmirrornormalize_cpu_vs_gpu():
+    def to_array(dali_out):
+        if isinstance(dali_out, TensorListGPU):
+            dali_out = dali_out.as_cpu()
+
+        return np.squeeze(dali_out.as_array())
+
+    def compare(val_1, val_2, reference=None):
```

Comment:
there are couple of functions to compare batches at the top of the file. I'd rather use those

Reply:
Done